src/gsy_e/gsy_e_core/sim_results/file_export_endpoints.py

```diff
@@ -306,9 +302,9 @@ def _water_tank_labels(self):
     @property
     def _pcm_tank_labels(self):
         labels = ["slot"]
-        for number, tank in enumerate(self.area.strategy.state.charger.tanks.tanks_states):
-            if isinstance(tank, PCMTankState):
-                tank_name_str = tank._params.name if tank._params.name else f"tank {number + 1}"
+        for number, tank in enumerate(self.area.strategy.state.get_results_dict()["tanks"]):
+            if tank["type"] == "PCM":
```

Comment:
I would prefer an enum here if we had more than 2 types, still ok though since these are just 2.